### PR TITLE
redland libraries

### PR DESCRIPTION
**Pull request by jeroenooms**
Redland is a scientific application framework which consists of 3 libraries: raptor, rasqal and librdf. Compare to:

 - [librdf0](https://packages.debian.org/stretch/librdf0) (Debian)
 - [redland](https://apps.fedoraproject.org/packages/redland/) (Fedora, EL)
 - [redland](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/redland.rb) (Mac Homebrew)

Thanks for reviewing!

### CODE REVIEW

File: mingw-w64-rasqal/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Maintainer: Jeroen Ooms <jeroenooms@gmail.com>
+
+_realname=rasqal
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=0.9.33
+pkgrel=1
+pkgdesc="RDF parser toolkit (mingw-w64)"
```
Comment (Alexpux) [inline mingw-w64-rasqal/PKGBUILD]: All packages have the same description?

**Comment by jeroenooms** [comment]
Sorry - fixed.

**Comment by jeroenooms** [comment]
Thanks!! Can you maybe help me build these packages for `repo.msys2.org`? I did the x64 one already myself but not had some trouble with my i686 installation.

**Comment by DavidEGrayson** [comment]
I thought @AlexPux built all the packages on http://repo.msys2.org.  Do you have some special arrangement with him?

**Pull request merged**